New Checkout Drop-in v1.4.0-beta2 (#273)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/index.mdx

Visit the [PaymentMethods](/dropins/checkout/containers/payment-methods) documentation for more information.

> The `ShippingMethods` container configuration structure has been modified. New properties are added to the `ShippingMethodsProps` interface:

Reviewer: ```suggestion
The configuration structure of the `ShippingMethods` container has changed and a new property was added to the `ShippingMethodsProps` interface:
```

> - **`ShippingMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as a selector for each shipping method. UI components available are: 'ToggleButton' and 'RadioButton'.

Reviewer: ```suggestion
- **`ShippingMethodsProps.UIComponentType` property added**: This property allows using a different UI component type as a selector for each shipping method. The available UI components are: `ToggleButton` and `RadioButton`.
```

> And `ShippingMethodsProps` interface is extending from `TitleProps` interface inheriting the following properties:

Reviewer: ```suggestion
The `ShippingMethodsProps` interface extends the `TitleProps` interface and inherits the following properties:
```
